docs/transition-phase-2.md

```diff
@@ -0,0 +1,72 @@
+# Transition your stake from phase-1 to phase-2 through CLI
+
+<!-- TODO -->
+
+
+>  **⚡ Note:** This program is a separate implementation from the staker program 
+>  used for Phase-1. All stakers are required to transition their stakes to this 
+>  program to participate in Phase-2.
+
+## Table of Contents
+- [Prerequisites](#prerequisites)
+- [1. Setup](#1-setup)
+- [2. Transition your stake](#2-transition-your-stake)
+
+## Prerequisites
+
+To transition your stake from phase-1 to phase-2, you need to have the following:
+-  Global parameter file as seen here https://github.com/babylonlabs-io/networks/blob/main/bbn-5/parameters/global-params.json that were used in phase-1
+-  The Bitcoin block height in which your staking transaction was included
+- Your `btc-staker` binary setup and running inclusive of the btc staker config 
+  properly setup
+
+## 1. Setup 
+
+First, you need to run the upgrade setup which configures the environment and 
+runs necessary upgrade procedures:
+
+```bash
+cd deployments/local
+make bbn-upgrade-v1
+```
+
+Next, you need to dump the default configuration file. This creates a template 
+that you'll customize with your specific values (like your Bitcoin block height 
+that your staking transaction was included in, wallet credentials, 
+and network details):
+
+```bash
+stakercli admin dump-config
+```
+
+This will create a `config.json` file in your current working directory. 
```

Comment:
we should instruct how can they configure to connect to the correct BTC chain and also wallet name on the BTC / cosmos